module1_solution/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ng-app="LunchCheck" lang="en">
+  <head>
+    <script type="text/javascript" src="../lib/angular.min.js"></script>
+    <script type="text/javascript" src="../lib/underscore-min.js"></script>
+    <script type="text/javascript" src="app.js"></script>
+    <link rel="stylesheet" href="styles/bootstrap.min.css">
+    <style>
+      .message { font-size: 1.3em; font-weight: bold; }
```

Comment:
move this out to a separate file and link like how you did bootstrap.min.css above